modules/ticket_selector/TicketSelector.php

```diff
@@ -13,42 +21,45 @@ abstract class TicketSelector
 {
 
     /**
-     * @var \EE_Event $event
+     * @var EE_Event
      */
     protected $event;
 
     /**
-     * @var \EE_Ticket[] $tickets
+     * @var EE_Ticket[]
      */
     protected $tickets;
 
     /**
-     * @var int max_attendees
+     * @var int
      */
     protected $max_attendees;
 
     /**
-     * @var array $template_args
+     * @var array
      */
     protected $template_args;
 
+    /**
+     * @var int
+     */
+    protected $ticket_rows = 0;
+
 
     /**
      * TicketSelectorSimple constructor.
      *
-     * @param \EE_Event    $event
-     * @param \EE_Ticket[] $tickets
+     * @param EE_Event    $event
+     * @param EE_Ticket[] $tickets
      * @param int          $max_attendees
      * @param array        $template_args
-     * @throws \EE_Error
      */
-    public function __construct(\EE_Event $event, array $tickets, $max_attendees, array $template_args)
+    public function __construct(EE_Event $event, array $tickets, $max_attendees, array $template_args)
     {
         $this->event = $event;
         $this->tickets = $tickets;
         $this->max_attendees = $max_attendees;
         $this->template_args = $template_args;
-        $this->template_args['hidden_inputs'] = $this->getHiddenInputs();
```

Comment:
moved this to just before we load the final template and after ticket rows have been generated (or removed via a filter) so that the new ticket rows counter is accurate